datajunction-server/datajunction_server/sql/parsing/backends/antlr4.py

```diff
@@ -287,12 +293,13 @@ def _(ctx: sbp.InlineTableContext):
     alias, columns = visit(ctx.tableAlias())
 
     # Generate default column aliases if they weren't specified
+    col_args = args[0] if isinstance(args[0], list) else args
```

Comment:
I'm unsure here but if this instead did `col_args = [args[0]] if not isinstance(args[0], list) else args[0]` to guarantee this is always a list, would that require less changes elsewhere?

Reply:
Actually I made a change slightly further down that simplified the changes in the ast:
```
        values=[[value] if not isinstance(value, list) else value for value in args],
```